// protoc --python_out=. replicas_response.proto; cp replicas_response_pb2.py client; cp replicas_response_pb2.py server

syntax = "proto3";

message Replica {
  string name = 1;
  string ip = 2;
  int32 port = 3;
  int32 chunk_id = 4;
  bool is_primary = 5;
}

message ReplicaList {
  bool success = 1;
  repeated Replica replicas = 2;
}